pkg/proc/core/linux_core_arm64.go

```diff
@@ -0,0 +1,350 @@
+package core
```

Comment:
I think that it should be possible to read arm64 core dumps on non-arm64 hosts, the name of this file should be linux_arm64_core.go so that it is compiled in unconditionally.

Also linux_amd64_core.go should not be renamed to linux_core_amd64.go.